ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/BackwardSyncAlgorithm.java

```diff
@@ -64,7 +64,10 @@ public CompletableFuture<Void> pickNextStep() {
       return executeSyncStep(firstHash.get())
           .thenAccept(
               result -> {
-                LOG.info("Backward sync target block is {}", result.toLogString());
+                LOG.atDebug()
```

Comment:
Was this intentionally moved from info to debug?

Reply:
Yes, the bws progress log is still not perfect and this could be noisy and not clear sometimes